app/grandchallenge/challenges/views.py

```diff
@@ -66,10 +79,10 @@ def get_context_data(self, *args, **kwargs):
                     (
                         "Here is an overview over the medical image analysis"
                         " challenges that have been hosted on Grand Challenge."
-                        "<br>Please <a href='{}'>contact us</a> if you would like "
-                        "to host your own challenge."
+                        "<br>Please fill in <a href='{}'>this form</a> "
+                        "if you would like to host your own challenge."
                     ),
-                    random_encode("mailto:support@grand-challenge.org"),
+                    random_encode(reverse("challenges:requests-create")),
```

Comment:
`random_encode` is not necessary here, this is only for making email addresses harder for spammers to scrape.